gitops/deployments/dev/loderunner.yaml

```diff
@@ -25,7 +25,7 @@ spec:
     spec:
       containers:
       - name: app
-        image: ghcr.io/retaildevcrews/ngsa-lr:beta-0.3.0-0323-0610-32
+        image: ghcr.io/retaildevcrews/ngsa-lr:beta-0.3.0-0325-0605-34
```

Comment:
I don't think Flux is going to allow me to roll back the version, as it will create a new commit with 5 minutes of me rolling it back. When this gets added to dev in the future Flux will automatically overwrite with the latest version, so I think this is fine to include

Reply:
I'll re-commit the actual version after its merged